src/Surfnet/StepupRa/RaBundle/Controller/VettingController.php

```diff
@@ -44,6 +44,9 @@ class VettingController extends Controller
      * @Template
      * @param Request $request
      * @return array|Response
+     *
+     * @SuppressWarnings(PHPMD.CyclomaticComplexity) Form validation and error handling
+     * @SuppressWarnings(PHPMD.NPathComplexity) Form validation and error handling
```

Comment:
Derp, I thought this was where we had all the hacky error handling as well. My issue here is that there's some technical debt growing here and that all this logic ought to be under test. If you agree, could you register this and document it after the `@SuppressWarnings`?